src/Zicht/Bundle/HtmldevBundle/Resources/views/styleguide/design-elements/typography.html.twig

```diff
@@ -0,0 +1,57 @@
+{% extends 'ZichtHtmldevBundle:styleguide:component.html.twig' %}
+{% import 'ZichtHtmldevBundle:macros:styleguide.html.twig' as styleguide %}
+
+
+{% block component_list %}
+    <div class="o-grid  o-grid--fixed">
+
+        <div class="o-grid__row  u-padding--t5">
+            <section class="o-grid__col  s-cms-content">
+
+                <h1>The quick brown fox jumps over the lazy dog.</h1>
+
+                <h2>Jived fox nymph grabs quick waltz?</h2>
+                <p class="introduction">
```

Comment:
Deze class komt nergens vandaan zover ik het nu zie, waar komt de styling van deze pagina vandaan?

Reply:
This is a copy-paste. I recognize it as a class from a project (introduction paragraph below title on content page probably). I'll remove it. It's not generic.